packages/algob/src/lib/status.ts

```diff
@@ -13,15 +13,14 @@ export async function balanceOf (
   deployer: Deployer,
   accountAddress: AccountAddress,
   assetID: number
-): Promise<modelsv2.AssetHolding | undefined> {
+): Promise<number|bigint> {
   const accountInfo = await deployer.algodClient.accountInformation(accountAddress).do();
   for (const asset of accountInfo.assets) {
     if (asset['asset-id'] === assetID) {
-      console.log("Asset Holding Info:", asset);
-      return asset;
+      return asset.amount;
     }
   }
-  return undefined;
+  return 0;
```

Comment:
this could be misleading, as the asset holding is not found but we return `0`. Maybe it makes sense to return `undefined` here?

Reply:
@robert-zaremba How do you think about @ratik21 suggestion?

Reply:
@ratik21 , balance should be a number (eg, if you have zero rubels and don't have rubel bank account then you still have 0, not undefined). I will add another function `getAssetHolding` which will follow the old behavior and return undefined if account didn't opt-in.